impl/integration/env.go

```diff
@@ -199,18 +199,18 @@ func NewEnv() (*Env, error) {
 	batchSize := 100
 	seq := sequencer.New(
 		sequencerClient,
-		logEnv.Log, mapEnv.Map, mapEnv.Admin,
-		domainStorage, mutations, queue, batchSize)
+		mapEnv.Admin,
+		domainStorage, mutations, batchSize)
 
-	d, err := domainStorage.Read(ctx, domainPB.DomainId, false)
-	if err != nil {
-		return nil, err
-	}
-	// NewReceiver will create a fist map revision right away.
-	receiver, err := seq.NewReceiver(ctx, d)
-	if err != nil {
-		return nil, fmt.Errorf("env: NewReceiver(): %v", err)
+	// RunBatchForAllDomains will create a fist map revision right away.
+	if err := seq.RunBatchForAllDomains(ctx); err != nil {
+		return nil, fmt.Errorf("env: RunBatchForAllDomains(): %v", err)
 	}
+	go func() {
```

Comment:
What stops this goroutine?

Reply:
Process exit -- ok that's bad. I'll fix it.